src/components/VideoCard/README.md

```diff
@@ -0,0 +1,244 @@
+
+### Body content
+
+Body content should be:
+
+- Actionable: start sentences with imperative verbs when telling merchants what actions are available to them, especially something new. Don’t use permissive language like “you can”.
+
+<!-- usagelist -->
+
+#### Do
+
+Get performance data for all of your sales channels.
+
+#### Don’t
+
+Now you can get performance data for all of your sales channels.
+
+<!-- end -->
+
+- Structured for merchant success: always put the most critical information
+  first
+- Clear: use the verb “need” to help merchants understand when they’re required
+  to do something
+
+<!-- usagelist -->
+
+#### Do
+
+To buy a shipping label, you need to enter the total weight of your shipment,
+including packaging.
+
+#### Don’t
+
+To buy a shipping label, you must enter the total weight of your shipment,
+including packaging.
+
+<!-- end -->
+
+### Call to action
+
+Buttons should be:
+
+Clear and predictable: merchants should be able to anticipate what will happen when they click a button. Never deceive merchants by mislabeling a button.
+
+<!-- usagelist -->
+
+#### Do
+
+Buy shipping label
+
+#### Don’t
+
+Buy
+
+<!-- end -->
+
+- Action-led: buttons should always lead with a strong verb that encourages
+  action. To provide enough context to merchants use the {verb}+{noun} format on
+  buttons except in the case of common actions like Save, Close, Cancel, or OK.
+
+<!-- usagelist -->
+
+#### Do
+
+View shipping settings
+
+#### Don’t
+
+View your settings
+
+<!-- end -->
+
+- Scannable: avoid unnecessary words and articles such as the, an, or a.
+
+<!-- usagelist -->
+
+#### Do
+
+Add menu item
+
+#### Don’t
+
+Add a menu item
+
+<!-- end -->
+
+### Thumbnail
+
+- Relevant: should describe the video through the use of images or be an image of a section in the video
+- Follow the 16:9 ratio, otherwise the image may appear cropped
+- If the thumbnail shows a person, avoid cropping the person’s head off
+
+---
+
+## Examples
+
+### Basic video card
+
+Use to surface educational information about a feature or opportunity in the context of where it is managed.
+
+```jsx
+<VideoCard
+  title="Getting Started"
+  primaryAction={[
+    {
+      content: 'Learn about getting started',
+      onAction: () => {},
+    },
+  ]}
+  description="Discover how Shopify can power up your entrepreneurial journey."
+  popoverActions={[{content: 'Dismiss', onAction: () => {}}]}
+  portrait
+>
+  <VideoThumbnail
+    videoLength={80}
+    thumbnailUrl="https://burst.shopifycdn.com/photos/smiling-businesswoman-in-office.jpg?width=1850"
+  />
+</VideoCard>
+```
+
+### Video card with secondary action
+
+Use when there are two distinct actions merchants can take on the information in the video.
+
+```jsx
+<VideoCard
+  title="Getting Started"
+  primaryAction={{
+    content: 'Learn about getting started',
+    onAction: () => {},
+  }}
+  secondaryAction={{
+    content: 'Click here',
```

Comment:
@krismausser @raydg-shopify is there a real example in the works or in production of a video card with a secondary action? If not, it's okay for there not to be an example for this prop, its shape is documented in the props table of the doc.

```suggestion
    content: 'Some real action here',
```